README.md

```diff
@@ -1,140 +1,38 @@
-Python packages since your last deployment, but if you aren't sure, it won't
-hurt to run it again.
-
-   ```bash
-   pipenv lock -r > requirements.txt
-   ```
-
-2. Create a new project on Heroku
-3. Under Resources click "Find more add-ons" and add the add on called "Heroku Postgres"
-4. Install the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-command-line)
-5. Run
-
-   ```bash
-   heroku login
-   ```
-
-6. Login to the heroku container registry
-
-   ```bash
-   heroku container:login
-   ```
-
-7. Update the `REACT_APP_BASE_URL` variable in the Dockerfile.
-   This should be the full URL of your Heroku app: i.e. "https://flask-react-aa.herokuapp.com"
-8. Push your docker container to heroku from the root directory of your project.
-   (If you are using an M1 mac, follow [these steps below](#for-m1-mac-users) instead, then continue on to step 9.)
-   This will build the Dockerfile and push the image to your heroku container registry.
-
-   ```bash
-   heroku container:push web -a {NAME_OF_HEROKU_APP}
-   ```
-
-9. Release your docker container to heroku
-
-      ```bash
-      heroku container:release web -a {NAME_OF_HEROKU_APP}
-      ```
-
-10. set up your database
-
-      ```bash
-      heroku run -a {NAME_OF_HEROKU_APP} flask db upgrade
-      heroku run -a {NAME_OF_HEROKU_APP} flask seed all
-      ```
-
-11. Under Settings find "Config Vars" and add any additional/secret .env
-variables.
-
-12. profit
-
-### For M1 Mac users
-
-(Replaces **Step 8**)
-
-1. Build image with linux platform for heroku servers. Replace
-{NAME_OF_HEROKU_APP} with your own tag:
-
-   ```bash=
-   docker buildx build --platform linux/amd64 -t {NAME_OF_HEROKU_APP} .
-   ```
-
-2. Tag your app with the url for your apps registry. Make sure to use the name
-of your Heroku app in the url and tag name:
-
-   ```bash=2
-   docker tag {NAME_OF_HEROKU_APP} registry.heroku.com/{NAME_OF_HEROKU_APP}/web
-   ```
-
-3. Use docker to push the image to the Heroku container registry:
-
-   ```bash=3
-   docker push registry.heroku.com/{NAME_OF_HEROKU_APP}/web
-   ```
-
-
-   "to see heroku errors"
-   ```
-   heroku logs -a <app-name>
-   ```
+# Zing
+
+## Summary
+[Zing](https://app-zing.herokuapp.com/) is a web application inspired by the popular messaging app, Slack. It gives users the ability to create a workspace that other users can join by invite. Users can then create a channel and message other users with live chat using websockets. Users can also search any user signed up on the application.
+
+Zing allows users to:
+* create an account
+* log into an existing account
+* log out of an account
+* create/edit/delete and view workspaces
+* create/edit/delete and view channels inside of workspaces
+* LIVE chat for each channel
+* able to search for specific users
+
+## Overall Structure
+### Back End
+The back end was built with [Flask](https://flask.palletsprojects.com/en/2.0.x/), and it utilizes a postgreSQL database.
+
+#### Database Diagram
```

Comment:
I dunno if the diagram is necessary here.

Reply:
Yeah I couldn't remember if it was a requirement or not for the readme even thought it's in the wiki.